core/src/main/scala/io/finch/route/RouterN.scala

```diff
@@ -41,9 +41,14 @@ trait RouterN[+A] { self =>
   def apply(route: Route): Option[(Route, A)]
 
   /**
+   * Attempts to match a route, but only returns any unmatched elements, not the value.
+   */
+  def exec(route: Route): Option[Route] = apply(route).map(_._1)
```

Comment:
Yeah. That would be a good improvement. Let's plan this for 0.8.0 then. How about to just make `exec` package-private for now?